test/hotspot/jtreg/gtest/GTestResultParser.java

```diff
@@ -57,7 +57,10 @@ public GTestResultParser(Path file) {
                             testCase = xmlReader.getAttributeValue("", "name");
                             break;
                         case "failure":
-                            failedTests.add(testSuite + "::" + testCase);
+                            String failedStr = testSuite + "::" + testCase;
+                            if (! failedTests.contains(failedStr)) {
```

Comment:
Please remove space between '! and failed..'.

Reply:
Thanks @lmesnik! I pushed the changes.